tesseract/tesseract_urdf/include/tesseract_urdf/geometry.h

```diff
@@ -184,17 +186,30 @@ inline tesseract_common::StatusCode::Ptr parse(std::vector<tesseract_geometry::G
   else if (geometry_type == "mesh")
   {
     std::vector<tesseract_geometry::Mesh::Ptr> meshes;
-    tesseract_common::StatusCode::Ptr status = parse(meshes, geometry, locator, visual);
+    tesseract_common::StatusCode::Ptr status = parse(meshes, geometry, locator, version, visual);
     if (!(*status))
       return std::make_shared<tesseract_common::StatusCode>(
           GeometryStatusCategory::ErrorParsingMesh, status_cat, status);
 
-    geometries = std::vector<tesseract_geometry::Geometry::Ptr>(meshes.begin(), meshes.end());
+    if (version < 2 && !visual)
+    {
+      for (const auto& mesh : meshes)
+      {
+        auto ch_verticies = std::make_shared<tesseract_common::VectorVector3d>();
```

Comment:
Spelling if that matters to you. It should be vertices

Reply:
Fixed.